src/components/PostsValues.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import { PostElement } from './PostElement'
+import { PostsContext } from './context/PostsContext'
+import IUser from './context/PostsContext'
+
+class PostsValues extends Component {
+  static contextType = PostsContext
+  context!: React.ContextType<typeof PostsContext>
+
+  render(): JSX.Element {
+    const { posts } = this.context
+
+    return (
+      <div>
+        {posts !== undefined && (
```

Comment:
Я уже писал) Что если у тебя будет posts undefined то в итоге ты отрендришь этот div

Можно сделать вот так, чтобы когда у тебя не существует постов, то ты ничего не рендрил

```suggestion
    if (!posts) {
      return null 
      }
    return (
      <div>
```